### PR TITLE
adding metrics for all anchor batch summary fields and key ratio

### PR DESCRIPTION
**Pull request by gvelez17**
# Anchor Batch Metrics - #PLAT-1156

## Description

Migrate ceramic-anchor-service to use Observability npm module and add metrics for anchor batches requested in PLAT-1156

## How Has This Been Tested?

Describe the tests that you ran to verify your changes. Provide instructions for reproduction.

- [ x] unit tests
- [x ] added tests to observability module

## Definition of Done

Before submitting this PR, please make sure:

- [ x] The work addresses the description and outcomes in the issue
- [ ] I have added relevant tests for new or updated functionality
- [x ] My code follows conventions, is well commented, and easy to understand
- [x ] My code builds and tests pass without any errors or warnings
- [x ] I have tagged the relevant reviewers
- [ ] I have updated the READMEs of affected packages
- [ ] I have made corresponding changes to the documentation
- [x ] The changes have been communicated to interested parties

## References:

Please list relevant documentation (e.g. tech specs, articles, related work etc.) relevant to this change, and note if the documentation has been updated.

### CODE REVIEW

**Comment by gvelez17** [comment]
oh sorry I didn't see the conflict!  fixing now

File: src/services/anchor-service.ts [hunk 1]
```diff
@@ -81,6 +80,11 @@ const logAnchorSummary = async (
     results
   )
 
+  Metrics.recordObjectFields('anchorBatch', anchorSummary)
+  Metrics.recordRatio('anchorBatch_failureRatio',
```
Comment (smrz2001) [inline src/services/anchor-service.ts:84]: @stbrody, Golda and I discussed this ratio as the right thing to capture for your request. Does this seem reasonable?

Golda's also recording more fields from the summary, which will probably be handy  to visualize trends for.

**Review by smrz2001 — APPROVED** [review]
LGTM 🚀 Would wait for @stbrody to confirm that we're measuring what we need.

File: src/services/anchor-service.ts (hunk 1 above)
Comment (stbrody) [reply src/services/anchor-service.ts:85]: yeah I think that seems right, thanks!

**Review by stbrody — APPROVED** [review]
lgtm

**Pull request merged**